src/app/platform_metadata.hpp

```diff
@@ -312,7 +327,7 @@ class PlatformMetadata {
     if (extension == "nds") {
       return PLATFORM_ID_NINTENDO_DS;
     }
-    if (extension == "md") {
+    if (extension == "md" || extension == "gen") {

```

Comment:
I'm not sure if there's a single preferred Genesis file extension, but a number of mine use .gen